.github/pull_request_template.md

```diff
@@ -0,0 +1,8 @@
+- [ ] Is the scope and purpose of the PR clear?
```

Comment:
Maybe this could be separated out into what the PR author should cover and what the reviewer should ask? (with some duplication maybe?)

Reply:
I tried to separate them out in https://github.com/illinois-ceesd/mirgecom/pull/506/commits/d01aea41259eb4ce849f9aa4b99ce3ef9948b453, but it seems to be a lot of duplication.

Reply:
Agree, not loving the duplication.

Reply:
Undid the duplication. This is ready for review @inducer.